get-started/install-golang.md

```diff
@@ -4,14 +4,14 @@ To build TiDB from source code, you need to install Go in your development envir
 
 ## TL;DR
 
-* Install Go 1.13 follow the [install instruction](https://golang.org/doc/install)
+* Install Go 1.16 follow the [install instruction](https://golang.org/doc/install)
 * Or manage Go toolchain using [gvm](https://github.com/moovweb/gvm)
 
-## Install Go 1.13
+## Install Go 1.16
 
 Most TiDB developers are using Linux or MacOS. If you are using Windows 10, to have a better experience, it is recommended to [install Windows Subsystem for Linux 2 \(WSL2\)](https://docs.microsoft.com/en-us/windows/wsl/install-win10) on Windows 10 first. TiDB should compile and run on Windows 10, but it is not expected to be deployed on Windows, where you might encounter many compatibility problems. In that case, you can either figure out the solutions on your own or share the problems on our [TiDB Internals forum](https://internals.tidb.io/).
 
-Currently, TiDB uses Go 1.13 to compile the code, but we plan to upgrade the compiler to Go 1.16 in the near future. To install Go 1.13, go to [Go's download page](https://golang.org/dl/), choose version 1.13, and then follow the [install instruction](https://golang.org/doc/install).
+Currently, TiDB uses Go 1.16 to compile the code, but we plan to upgrade the compiler to Go 1.16 in the near future. To install Go 1.16, go to [Go's download page](https://golang.org/dl/), choose version 1.16, and then follow the [install instruction](https://golang.org/doc/install).
```

Comment:
s/but we plan to upgrade the compiler to Go 1.16 in the near future//

Reply:
```suggestion
Currently, TiDB uses Go 1.16 to compile the code. To install Go 1.16, go to [Go's download page](https://golang.org/dl/), choose version 1.16, and then follow the [install instruction](https://golang.org/doc/install).
```

Reply:
Thanks!